src/Mono.Android/Android.App/SyncContext.cs

```diff
@@ -14,7 +15,15 @@ public override SynchronizationContext CreateCopy ()
 
 		public override void Post (SendOrPostCallback d, object state)
 		{
-			using (Handler h = new Handler(Application.Context.MainLooper)) {
+			var looper = Application.Context?.MainLooper;
+			if (looper == null) {
+				var message = "No Looper available to this SynchronizationContext, callback will not be executed.";
+				if (JNIEnv.IsRunningOnDesktop)
+					message += " Using await when running on the Android Designer is not currently supported.";
```

Comment:
message += " Using `await` when running on the Android Designer is not currently supported. " +
        "Please use the `View.IsInEditMode` property";

Provide a pointer for how to check when the designer is running, so that they can have a better experience.